packages-exp/auth-exp/src/core/user/token_manager.test.ts

```diff
@@ -110,4 +111,40 @@ describe('core/user/token_manager', () => {
       expect(tokens.refreshToken).to.eq('refresh');
     });
   });
+
+  describe('fromPlainObject', () => {
+    const fpo = (obj: { [key: string]: unknown }) =>
```

Comment:
should we name this type? you use it in the implementation as well

Reply:
also why do we need this alias?

Reply:
Dropped the alias, added that thing as a type: "PersistedBlob"